### PR TITLE
Update Android.gitignore

### PR DESCRIPTION
**Pull request by MaTriXy**
**Reasons for making this change:**

I've added a few missing ignores to the file .

Added missing configurations for Android studio

### CODE REVIEW

File: Android.gitignore [hunk 1]
```diff
@@ -32,6 +51,15 @@ proguard/
 # Android Studio captures folder
 captures/
 
+# OS-specific files
```
Comment (shiftkey) [inline Android.gitignore]: We're tracking these in `Global/OSX.gitignore` - please drop these...

Comment (MaTriXy) [inline Android.gitignore]: ok will update now.

**Comment by tuxayo** [comment]
It seems that part of those changes are covered by the [global JetBrains.gitignore](https://github.com/github/gitignore/blob/c74eb94334463252a15c8f80876847cc580b8dd3/Global/JetBrains.gitignore).

So the real problem is that there doesn't seem to be a clear indication that someone coming just for a .gitignore for Android should also use JetBrains.gitignore (and maybe others??)

**Comment by MaTriXy** [comment]
Android Studio has some similarity with JetBrains. 
When someone wants an Android .gitignore he will most likely use Android studio (at minimum) so having a few overlapping rules is ok as it is usually the only IDE for Android in a specific computer.

**Pull request closed without merging**